rotkehlchen/tests/external_apis/test_etherscan.py

```diff
@@ -45,7 +45,7 @@ def _handle_killed_greenlets(greenlet: gevent.Greenlet) -> None:
 
 
 @pytest.mark.skipif(
-    'TRAVIS' in os.environ,
+    'CI' in os.environ,
```

Comment:
Guess you checked and it does but this is both in Circle and Travis, right?

Reply:
Yes CI is by default true on travis